architecture/core_objects/builds.adoc

```diff
@@ -172,41 +174,19 @@ can also add authentication entries to this file by running the `docker login`
 command. The file will be created if it does not exist.
 
 Kubernetes provides the
-https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/secrets.md[Secret]
-resource, which is used to store your configuration and passwords. You must
-first create the `*Secret*` before builds can use your *_.dockercfg_* file for
-pushing the output image:
```

Comment:
@bparees so we have nothing to document here before it gets done... once you will be able to add new dockercfg to builder account, then this 'manual' way will be obsoleted and you just add the secrets and you don't have to specify anything in BuildConfig